tests/time_tests/test_runner/conftest.py

```diff
@@ -161,40 +175,155 @@ def validate_test_case(request, test_info):
     """Fixture for validating test case on correctness.
 
     Fixture checks current test case contains all fields required for
-    a correct work. To submit results to a database test case have
-    contain several additional properties.
+    a correct work.
```

Comment:
In next PR try to get rid of extra schemas by adding some data structure to freeze and validate fields of test case